src/app/shared/Carousel/index.js

```diff
@@ -4,22 +4,42 @@ import './Carousel.scss';
 class Carousel extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = { buttonsVisibility: false };
+
     this.carouselRef = React.createRef();
+    this.prevButtonRef = React.createRef();
+    this.nextButtonRef = React.createRef();
     this.wrapperRef = React.createRef();
+
     this.isTouchDevice = 'ontouchstart' in window || navigator.msMaxTouchPoints;
 
     this.carouselStyle = {
       translation: 0,
       scrollCounter: 0,
       doubleSideMargin: 30
     };
-
     this.carouselStyleSheet = { transform: null, width: null };
-
     this.wrapperStyle = { overflowX: this.isTouchDevice ? 'scroll' : 'hidden' };
+
+    this.updateButtonsAppearance = this.updateButtonsAppearance.bind(this);
   }
 
+  updateButtonsAppearance = () => {
+    if (this.carouselStyle.translation === 0) {
+      this.prevButtonRef.current.setAttribute('disabled', true);
+    } else {
+      this.prevButtonRef.current.removeAttribute('disabled');
+    }
+    if (
+      this.carouselStyle.width - this.carouselStyle.wrapperWidth <=
```

Comment:
use parentheses for `this.carouselStyle.width - this.carouselStyle.wrapperWidth` expression